src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ManageContent/ContentController.cs

```diff
@@ -151,24 +151,33 @@ public async Task<ActionResult<List<DataBlock>>> GetAvailableDataBlocks(Guid rel
             Guid releaseId, Guid contentSectionId, Guid contentBlockId, CommentSaveRequest saveRequest)
         {
             return await _contentService
-                .AddCommentAsync(releaseId, contentSectionId, contentBlockId, saveRequest)
+                .AddComment(releaseId, contentSectionId, contentBlockId, saveRequest)
                 .HandleFailuresOrOk();
         }
 
         [HttpPut("comment/{commentId}")]
-        public async Task<ActionResult<CommentViewModel>> UpdateComment(Guid commentId, CommentSaveRequest saveRequest)
+        public async Task<ActionResult<CommentViewModel>> UpdateComment(Guid commentId, 
+            CommentSaveRequest saveRequest,
+            [FromQuery(Name = "resolved")] bool? resolved = null)
         {
+            if (resolved.HasValue)
```

Comment:
Is it possible to update other values whilst resolving?  Not sure what the front end looks like

Reply:
You can't. The resolve button appears even if you aren't currently editing the comment.